### ISSUE TITLE
Unexpected result with command 'quantitative' when send exact payload

### ISSUE THREAD

**Issue by airween** [coreruleset/go-ftw#381]
I tried to run `ftw quantitative` with 100K corpus on PL 4. In the result I saw rules 920273 and 921170 with 100000 matches which means all pattern matches with the rule.

Then I cut a pattern and tried to run only that one with `-p` (payload string):
```
$ ./ftw quantitative -p "Above all, we need an effective system of external and independent monitoring." -P 4 -d ../coreruleset/
⌛  Running quantitative tests
No false positives detected with the passed corpus
```
So it showed no matches, which is unexpected after the previous report.

**Comment by fzipi** [comment]
This was fixed in f5a9644fb3c169d0a0078882f13205f898bb79c2.

We need to push a new version.
